examples/inference-deployments/mpt/mpt_30b.yaml

```diff
@@ -0,0 +1,25 @@
+name: mpt-30b-ft
+compute:
+  gpus: 1
+  instance: oci.vm.gpu.a10.1
+image: mosaicml/inference:0.1.1
+replicas: 1
+command: |
+  export PYTHONPATH=/code/llm-foundry:/code/examples:/code
+integrations:
+- integration_type: git_repo
+  git_repo: mosaicml/examples
+  git_commit: dcc4c4dff002d282a2f432ccfcab4b61dc6a53af
+- integration_type: git_repo
+  git_repo: mosaicml/llm-foundry
+  git_commit: 2c92faa5ce31888214bdb582ac7f5756d0d3dacd
+  ssh_clone: false
```

Comment:
Double check both this and the other commit hash before merging - make sure it completely works